R/make_linter_from_regex.R

```diff
@@ -34,12 +34,12 @@ make_linter_from_regex <- function(regex,
 
 #' Determine if a regex match is covered by an expression in a source_expression
 #'
-#' @param   match   The position where a regex match was observed.
-#'   match must have entries "start", "end", and "line_number".
-#' @param   source_expression   A source_expression
-#' @param   token_type    Restrict analysis to tokens of this type, for example,
-#'   with token_type = "STR_CONST" you can check that a regex match occurs
-#'   within a string
+#' @param match The position where a regex match was observed.
+#'   Possible values: `"start"`, `"end"`, and `"line_number"`.
+#' @param source_expression A source_expression.
+#' @param token_type Restrict analysis to tokens of this type, for example,
+#'   with `token_type = "STR_CONST"` you can check that a regex match occurs
+#'   within a string.
```

Comment:
Not related, but minor docs clean-up. Didn't want to create a separate PR just for this change.